### PR TITLE
#1624 bids_MergeNifti PLD vector

### LINKED ISSUE #1624: Import of multi-sequence GE data
### Description

Import of GE data works automatically as long as data from a single sequence is provided per session. In case that DICOMs from multiple sequences are mixed in a single folder, ExploreASL won't be able to automatically merge them and get the PLDs and TRs etc. 

We have to use the MCW data for this case, mix them all together and read jointly all the data and make sure that a proper vector of PLDs and TRs and LabelingDurations is created and saved. Because deltaM/m0Scan differentiation works nicely even for multi-sequence merged data.

This has been configured and fixed in GE data from MCW. So we only need to verify that this works, check it and add to flavors.

### Tasks

- [x] Fix import for GE_PCASL_3Dspiral_RX28.0_multiPLD_1
- [x] Add to flavors
- [x] Update older flavors with a proper PLD vector 
- [x]   {'GE_PCASL_3Dspiral_DV22.0_1'     }    {'Both'}    {'Different file content'}    {'Different file content: /sub-Sub1/perf/sub-Sub1_asl.json'                }
- [x]     {'GE_PCASL_3Dspiral_DV23.1_1'     }    {'Both'}    {'Different file content'}    {'Different file content: /sub-11/perf/sub-11_asl.json'                    }
- [x]     {'GE_PCASL_3Dspiral_DV24.0_1'     }    {'Both'}    {'Different file content'}    {'Different file content: /sub-Sub103/perf/sub-Sub103_asl.json'            }
- [x]     {'GE_PCASL_3Dspiral_DV25.0_1.5T_1'}    {'Both'}    {'Different file content'}    {'Different file content: /sub-D21002/perf/sub-D21002_asl.json'            }
- [x]     {'GE_PCASL_3Dspiral_RX28.02_1'    }    {'Both'}    {'Different file content'}    {'Different file content: /sub-Sub124/perf/sub-Sub124_asl.json'            }

### Release notes

Read GE mixture of single-PLDs as a multi-PLD sequence.

### PR DESCRIPTION
**Pull request by jan-petr**
### Linked issue

Close #1624

### CODE REVIEW

**Review by HenkMutsaerts — APPROVED** [review]
Would be nice if you can add a bit more comments to explain what you're doing why. Otherwise it looks fine

File: Modules/Module_Import/NII2BIDS/xASL_bids_MergeNifti.m [hunk 1]
```diff
@@ -688,20 +688,37 @@
...
-				fieldsDuplicityCheck = {'GELabelingDuration','InversionTime','LabelingDuration'};
-				for iField = 1:length(fieldsDuplicityCheck)
-					if isfield(outputJSON, fieldsDuplicityCheck{iField}) && isfield(currentJSON, fieldsDuplicityCheck{iField})
-						if ~isequal(outputJSON.(fieldsDuplicityCheck{iField}), currentJSON.(fieldsDuplicityCheck{iField}))
-							warning('Difference in field %s between merged JSONs', fieldsDuplicityCheck{iField});
+				bExportVector = 0;
+				% For certain parameters - gather the fields from merged JSONs as a vector
+				fieldsExportVector = {'PostLabelDelay'};
+				for iField = 1:length(fieldsExportVector)
+					if strcmp(listFieldNames{iFieldName}, fieldsExportVector{iField}) && isfield(currentJSON, listFieldNames{iFieldName})
```
Comment (HenkMutsaerts) [inline Modules/Module_Import/NII2BIDS/xASL_bids_MergeNifti.m]: You could do strcmpi

**Review by HenkMutsaerts — COMMENTED** [review]
Minor comment

File: Modules/Module_Import/NII2BIDS/xASL_bids_MergeNifti.m (hunk 1 above)
Comment (jan-petr) [reply Modules/Module_Import/NII2BIDS/xASL_bids_MergeNifti.m]: Done.

**Pull request merged**